datahub-web-react/src/app/entityV2/document/DocumentNativeProfile.tsx

```diff
@@ -91,6 +98,22 @@ const sidebarSections = [
 export const DocumentNativeProfile: React.FC<Props> = ({ urn, document, loading = false, refetch }) => {
     const [sidebarClosed, setSidebarClosed] = useState(true); // Start closed by default
     const isCompact = React.useContext(CompactContext);
+    const { getRootNodes } = useDocumentTree();
+    const history = useHistory();
+    const entityRegistry = useEntityRegistry();
+
+    const handleDelete = React.useCallback(
```

Comment:
We are adding support for deleting a document directly from within the profile..